Update getting-started-with-entity-framework-core.md (#7240)
umbraco/UmbracoDocs: 16/umbraco-cms/tutorials/getting-started-with-entity-framework-core.md

## Prerequisite

* An Umbraco project with content

> * If code created in separate project from Umbraco package, project has to reference:
>   * Microsoft.EntityFrameworkCore
>   * Microsoft.EntityFrameworkCore.Relational

Reviewer: ```suggestion
* If the code is created in a separate project (outside the main Umbraco project), that project must reference:
  * `Microsoft.EntityFrameworkCore`
  * `Microsoft.EntityFrameworkCore.Relational`
```
